Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableBus.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzFramework/Spawnable/SpawnableEntitiesInterface.h>
+
+namespace AzFramework
+{
+    // Provides spawn notifications from Spawnable API
+    class SpawnableNotifications
+        : public AZ::EBusTraits
+    {
+    public:
+        // EBusTraits overrides
+        using BusIdType = EntitySpawnTicket::Id;
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+
+        //! Notifies that spawn operation was completed
+        virtual void OnSpawn(
+            [[maybe_unused]] EntitySpawnTicket spawnTicket,
+            [[maybe_unused]] AZStd::vector<AZ::EntityId> entityList) {}
+
+        //! Notifies that despawn operation was completed
+        virtual void OnDespawn([[maybe_unused]] EntitySpawnTicket spawnTicket) {}
```

Comment:
```suggestion
        virtual void OnDespawn([[maybe_unused]] const EntitySpawnTicket& spawnTicket) {}
```

Reply:
I can't make it const because SpawnableAPI requires ticket to be non-const. I can pass it by reference though? Not sure how much that matters, since I am capturing it in lambda for async callback later.